app/bundles/CoreBundle/Event/EntityExportEvent.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Mautic\CoreBundle\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class EntityExportEvent extends Event
```

Comment:
Please make new classes final by default. If a class isn't final it should have a good reason not to be.
```suggestion
final class EntityExportEvent extends Event
```

Reply:
fixed